packages/@lwc/template-compiler/src/__tests__/fixtures/scoped-slots/slot-content-with-directives/invalid/if-block/actual.html

```diff
@@ -0,0 +1,17 @@
+<template>
+    <x-child>
+        <template lwc:slot-data="variations" slot="foo">
+            <template lwc:if={variations.variation1}>
+                Button 1
+                <x-button1></x-button1>
+            </template>
+            <template lwc:elseif={variations.variation2}>
+                <!-- Button 2 -->
+                <x-button2></x-button2>
+            </template>
+            <template lwc:else>
+                <x-button3></x-button3>
```

Comment:
Should this be a comment or text node, just to make sure the error works in `lwc:else`?